Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -197,7 +156,8 @@ function PostDiscussionThreads
 #
 function FetchActiveDiscussionThreads
 {
-    $threadsDictionary = [PsWorkarounds.Helper]::GetThreadsDictionary($script:discussionClient, $script:artifactUri)
+    $request = $script:discussionClient.GetThreadsAsync($artifactUri, $null, [System.Threading.CancellationToken]::None)
```

Comment:
General question about cancellation - is it an explicit decision not to cancel? what about if the user abandoned the PR?
A side note: not cancelling might impose a security hole in a form of denial of service although I'm not familiar enough with the code/platform to actually come up a scenario in which this would happen.